src/ScrollingBookView.ts

```diff
@@ -11,19 +11,23 @@ export default class ScrollingBookView implements BookView {
         this.iframe = iframe;
     }
 
-    public setTopMargin(topMargin: number): void {
-        this.topMargin = topMargin;
+    public setSideMargin(margin: number) {
+        this.sideMargin = margin;
```

Comment:
You probably know this, but there is not a need to have private instance variables with public setters like this in TypeScript. Instead of this:

```typescript
private sideMargin: number = 0;

public setSideMargin(margin: number) {
    this.sideMargin = margin;
}
```

I would just do this:

```typescript
sideMargin: number = 0;
```

If you then later decide you want to have the setter (or getter!) perform some action, you can refactor the code as such without breaking the public API:

```typescript
private _sideMargin: number = 0;

get sideMargin(): number {
     // Do whatever here!
    return this._sideMargin;
}

set sideMargin(margin: number) {
    this._sidemargin = margin;
    // Do whatever here!
}
```

The advantages of doing this way are that is avoids the extra code until you're sure you need it and it lets people use property syntax (e.g. `scrollingBookview.sideMargin = 8`) for getters and setters.